### PR TITLE
SEO-187924-ASP.NET-Web-Forms-RichTextEditor-Validation-HTTP-to-HTPPS

### PR DESCRIPTION
**Pull request by Faith-Atieno**
Title | Description
-- | --
|Task Category | Site Audit work|
|Content Task Link/Mail Screenshot | NA|
|Hotfix |https://github.com/syncfusion-content/asp.netwebforms-docs/pull/548|
|Ticket/Task link |https://dev.azure.com/Syncfusion-Infrastructure/Syncfusion%20SEO/_workitems/edit/187924|
|Work link |https://syncfusion.sharepoint.com/:x:/r/sites/SEOWebsiteAuditing/_layouts/15/Doc.aspx?sourcedoc=%7B0C07BBB1-1233-47A7-818C-275A98FD0063%7D&file=Faith-%20Site%20Audit.xlsx&action=default&mobileredirect=true|

|Changes Made | Reason for change |
|-- | -- |
|Changed HTTP links to HTTPS | To avoid redirect links and to improve performance of the page |

### CODE REVIEW

File: aspnet/RichTextEditor/Validation.md [hunk 1]
```diff
@@ -1,15 +1,15 @@
 ---
 layout: post
 title: Validation in RichTextEditor | Syncfusion | ASP.NET Webform
-description: Validation to format the RichTextEditor control's content
+description: This article explains more about Validation to format the ASP.NET Web Forms RichTextEditor control's content
```
Comment (Yvone-Atieno) [inline aspnet/RichTextEditor/Validation.md:4]: Check description

File: aspnet/RichTextEditor/Validation.md [hunk 2]
```diff
@@ -1,15 +1,15 @@
...
 platform: aspnet
 control: RTE
 documentation: ug
 keywords: RichTextEditor, Validation, JQuery Validation, ASP Validation
 
 ---
 
-# Validation 
+# ASP.NET Web Forms RichTextEditor Validation
```
Comment (Yvone-Atieno) [inline aspnet/RichTextEditor/Validation.md:12]: Check H1

**Review by Yvone-Atieno — CHANGES_REQUESTED** [review]
Commented

File: aspnet/RichTextEditor/Validation.md (hunk 1 above)
Comment (Faith-Atieno) [reply aspnet/RichTextEditor/Validation.md:4]: resolved

File: aspnet/RichTextEditor/Validation.md (hunk 2 above)
Comment (Faith-Atieno) [reply aspnet/RichTextEditor/Validation.md:12]: resolved

File: aspnet/RichTextEditor/Validation.md (hunk 1 above)
Comment (Yvone-Atieno) [inline aspnet/RichTextEditor/Validation.md:4]: Please check description

**Review by Yvone-Atieno — CHANGES_REQUESTED** [review]
Commented

File: aspnet/RichTextEditor/Validation.md (hunk 1 above)
Comment (Yvone-Atieno) [inline aspnet/RichTextEditor/Validation.md:4]: Updated title and description proper, this is not an article

File: aspnet/RichTextEditor/Validation.md (hunk 2 above)
Comment (Yvone-Atieno) [inline aspnet/RichTextEditor/Validation.md:12]: Update H1 as Validation  in  ASP.NET Web Forms RichTextEditor

File: aspnet/RichTextEditor/Working-with-Tables.md [hunk 3]
```diff
@@ -1,20 +1,20 @@
 ---
 layout: post
-title: Working with Tables opreation | RTE | Syncfusion | ASP.NET
-description: Working with Tables customization for RichTextEditor control
+title: Working with Tables in ASP.NET Web Forms RichTextEditor | Syncfusion
+description: Learn here all about working with Tables in Syncfusion ASP.NET Web Forms RichTextEditor control, its elements, and more.
```
Comment (Yvone-Atieno) [inline aspnet/RichTextEditor/Working-with-Tables.md:4]: Learn here all about working with tables in Syncfusion ASP.NET Web Forms RichTextEditor control, its elements, and more.

**Review by Yvone-Atieno — CHANGES_REQUESTED** [review]
Commented

**Review by Yvone-Atieno — CHANGES_REQUESTED** [review]
Commented

**Review by Yvone-Atieno — APPROVED** [review]
Approved

**Review by Aishwarya-Ganesan — APPROVED** [review]
fine.

**Pull request merged**